relay-common/src/project.rs

```diff
@@ -1,3 +1,11 @@
+//! This module contains [`ProjectKey`] and [`ProjectId`] types and necessary traits
+//! implementations.
+//!
+//! - [`ProjectId`] is just a wrapper over `u64` and should be considered as implementations details,
+//! as it can change in the future,
```

Comment:
Q: is it worth mentioning that something can change in the future? Isn't all code subject to that? I'd only mention the first part:
```suggestion
//! - [`ProjectId`] is a wrapper over `u64`.
```
I'd love to see what you think, not sure if I'm missing something.